frontend/src/routes/routes.js

```diff
@@ -68,7 +68,7 @@ const routes = [
     },
   },
   {
-    path: '/community/:tab',
+    path: '/community/:tab/:page?',
```

Comment:
i would prefer the same renaming for the path to express more that it has nothing to do with 'gradido communities' but with 'contributions inside the local community'?
I was very confused in the first run...